### PR TITLE
Fix fellowship button

### PR DESCRIPTION
**Pull request by derrekchow**
Please test by making the height of the window somewhat short and compare with the live site. This will now work for reasonably sized heights.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules/
-nvm
\ No newline at end of file
+nvm
+package-lock.json
```
Comment (nikhilro) [inline .gitignore]: `package-lock.json` needs to be in the git repository

File: coalition/index.html [hunk 2]
```diff
@@ -88,37 +88,37 @@
...
+					<div class="col-10 col-md-4">
+						<h1>The Impact Coalition</h1>
+						<p class="lead">
+							We match you with a company that truly makes a difference.
+							<br/>
+							<br/>  
+							Applications for The 2019 Impact Coalition are now closed. <a target="_blank" href="https://impactlabs.us19.list-manage.com/subscribe/post?u=e271c51cea784ba1c111d990e&amp;id=90db606254">Sign up for our newsletter</a> to find out when they open for 2020!
+						</p>
+					</div>
+					<div class="col-1 col-md-2"></div>
```
Comment (nikhilro) [inline coalition/index.html]: not immediately clear what's this for

File: fellowship/index.html [hunk 3]
```diff
@@ -123,31 +123,31 @@ <h1 class="text-primary">The Impact Fellowship</h1>
...
+				<div class="row">
+					<div class="col-0 col-md-4"></div>
+					<div class="col-12 col-md-4">
+					<h2>About the Fellowship</h2>
+					<p class="lead">
+						The Impact Fellowship is a 2-week gathering of talented and passionate computer science students who want to use their skills for social good!
+					</p>
+					</div>
+					<div class="col-12 col-md-3 mt-5 text-center">
+						<img alt="Abou the Fellowship" width="350px" src="../assets/illustrations/about_the_fellowship.svg" />
```
Comment (nikhilro) [inline fellowship/index.html]: not part of this pr but typo on "about the fellowship"

File: index.html [hunk 4]
```diff
@@ -87,7 +87,7 @@
 		</div>
 		</nav>
 
-		<header data-aos="fade-up">
+		<header data-aos="fade-up" style="height:100vh">
```
Comment (nikhilro) [inline index.html]: why do we need this inline and in `main.scss`

Comment (derrekchow) [reply index.html]: It's not gonna overwrite if I do it in `main`, I think its better here since its a single exception to all the other headers on the other pages.

File: fellowship/index.html (hunk 3 above)
Comment (derrekchow) [reply fellowship/index.html]: Good catch

File: index.html (hunk 4 above)
Comment (nikhilro) [reply index.html]: I would prefer if give this a class name. use css selector in main.scss to specify this with a comment on why this header is special

File: coalition/index.html (hunk 2 above)
Comment (derrekchow) [reply coalition/index.html]: It adds the proper column spacing, or else the text would stretch the whole width of the screen.

File: index.html (hunk 4 above)
Comment (derrekchow) [reply index.html]: It's not really special enough to be its own class, it just happens the height should be 100vh and not 130vh for padding. `main.scss` should be used for styles that apply to most tags, many elements. This specific kind of thing should be done inline with bootstrap so I've changed it.

Comment (derrekchow) [reply index.html]: Mb, look at the change now. I should use inline bootstrap not custom styles.

Comment (nikhilro) [reply index.html]: thanks!
